pandas/tests/test_frame.py

```diff
@@ -7693,6 +7693,10 @@ def test_info_memory_usage(self):
         DataFrame(1,index=pd.MultiIndex.from_product([['a'],range(1000)]),columns=['A']).index.nbytes
         DataFrame(1,index=pd.MultiIndex.from_product([['a'],range(1000)]),columns=['A']).index.values.nbytes
 
+        # sys.getsizeof works the same as memory_usage with defaults, albeit with some
+        # GC overhead
+        self.assertAlmostEqual(sys.getsizeof(df), df.memory_usage().sum(), delta=100)
```

Comment:
test this on ``series/Index/Categorical`` as well. (for ``Index`` put a test in test_index on the ``Base`` which will run for all indexes).